src/assets/scss/base/inputs.scss

```diff
@@ -0,0 +1,33 @@
+.inputgroup {
```

Comment:
Bitte CSS auslagern in scss File.

Reply:
Hier müssten bitte noch die CSS Selektoren durch BEM ersetzt werden.

Mein Vorschlag für das Schema:

InputGroup -> Block
InputGroup__Label / InputGroup__TextArea -> Element
InputGroup__Radio--round -> Modifier